modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc

```diff
@@ -76,6 +77,32 @@ A clone of the Git repository is present in the filesystem of the new workspace.
 +
 The workspace has a unique URL: `pass:c,a,q[{prod-url}]#workspace__<unique_url>__`.
 
+.. Using a direct URL to `devfile`:
```

Comment:
FYI, this step is now in the wrong place: it's currently placed after the workspace has been started.

I also think this step makes the procedure difficult to follow and less readable.

In my personal opinion, the best solution to this update is to turn this page file into an assembly with two procedures in two separate `proc_...` files as follows:
1. Start a new branch from `main` (not from the branch of this PR).
2. In the new assembly file, keep the original heading and original file name (`starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc`), the original three intro paragraphs, and move up to them both the original bottom tip about browser bookmark managers and original *Additional resources*.
3. You can think of some parallel headings for two new procedure files, e.g.:
```
= Starting a new workspace from a Git-repository URL
= Starting a new workspace from a devfile URL
```
4. Under *Additional resources* in the assembly file, add two include statements for the two `proc_...` files.
5. Introduce the two new procedures in the intro of the assembly file as unordered list items.
6. Move the original content starting from the first tip (*"You can also use the Git Repo URL ..."*) up to and including *"The workspace has a unique URL"* to a new `proc_...` file that has the heading like `= Starting a new workspace from a Git-repository URL`.
7. There, consider moving the following three sentences into a new *Verification* section as three separate list items (will be the same for both procedures):
```
After you enter the URL to start a new workspace in a browser tab, it renders the workspace-starting page.

When the new workspace is ready, the workspace IDE loads in the browser tab.

A clone of the Git repository is present in the filesystem of the new workspace.
```
8. There, consider turning the following sentence into a NOTE (will be the same for both procedures):
```
The workspace has a unique URL: https://<che_fqdn>#workspace<unique_url>.
```
9. After you finalize the first procedure file, copy all its content into the second procedure file, and there incorporate the new content from this PR that's currently in step 2b) *"Using a direct URL to devfile:"* and as related to it elsewhere.

Reply:
thanks for the input, I was thinking along similar lines but didn't have as precise a picture in my head as you just described.